projects/dark-matter-reana.yml

```diff
@@ -0,0 +1,37 @@
+---
+name: REANA workflow for Dark Matter Searches
+postdate: 2023-02-14
+categories:
+  - Analysis tools
+durations:
+  - 3 months
+skillset:
+  - Python
+  - Docker
+status:
+  - Available
+project:
+  - IRIS-HEP
+experiments:
+  - SuperCDMS
```

Comment:
This is listed as SuperCDMS as that's @pibion's experimental affiliation. This can be revised to something else if perferable.

Reply:
We should probably list this as RCFM, @besk467 is the project lead on this.  I'm hoping we'll eventually get a SuperCDMS project in here, but I need to do some more recruiting/data registration before that's likely.

Reply:
@pibion @besk467 Is there a website for RCFM? Googling isn't turning up anything. If not I can just link to https://github.com/Cisneros-Galaxy/RCFM.

Reply:
I guess I've not understood what RCFM is..can someone elaborate?

Reply:
RCFM is the name of a galaxy rotation curve fitting experiment. @pibion and @besk467 are contacts for it and can supply more information / web info for it.

Reply:
experiment or model? (I find only model as being known to google)

Reply:
@besk467 Is there an experimental collaboration that this work is associated with?

Reply:
@matthewfeickert the RCFM runs on a large public library of rotation curve data, http://astroweb.cwru.edu/SPARC/, which itself is a collection of high quality rotation curves with associated photometry for the mass models.

Reply:
Okay so seems like there's no experimental collaboration that you or @pibion are affiliated with that's relevant for this project. That's fine, but this field should just be left blank then and we need to update the text in `description`.

Reply:
Oh. @davidlange6 this is going to cause YAML linting errors.

```console
$ python _scripts/check_projects.py 
Hi there. Checking your projects
Missing expected attribute
./projects/dark-matter-reana.yml
experiments
```

how do you want to handle this? Create a generic "Dark Matter Detection" category in `project_metadata.yml`?

Reply:
I believe I used "Any" as the appropriate entry in this case (at least that is what the template tells me I did). I'll make sure that has the desired behavior when merged - the intent was not to force things to have an "experiment" specified when it doesn't make sense.

Reply:
Great thanks!

Reply:
@besk467 is the expert on this but I think an even broader "Dark Matter" category would be appropriate here, the RCFM analysis is looking at fitting rotation curves and I'd classify it as examining the evidence for dark matter.